<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialOpsWidget</class>
 <widget class="QWidget" name="SerialOpsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="portLayout">
      <item>
        <widget class="QComboBox" name="comboPort"/>
      </item>
      <item>
        <widget class="QComboBox" name="comboBaud"/>
      </item>
      <item>
        <widget class="QComboBox" name="comboDataBits">
         <item>
          <property name="text"><string>8</string></property>
         </item>
         <item>
          <property name="text"><string>7</string></property>
         </item>
         <item>
          <property name="text"><string>6</string></property>
         </item>
         <item>
          <property name="text"><string>5</string></property>
         </item>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="comboParity">
         <item>
          <property name="text"><string>None</string></property>
         </item>
         <item>
          <property name="text"><string>Even</string></property>
         </item>
         <item>
          <property name="text"><string>Odd</string></property>
         </item>
         <item>
          <property name="text"><string>Space</string></property>
         </item>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="comboStopBits">
         <item>
          <property name="text"><string>1</string></property>
         </item>
         <item>
          <property name="text"><string>1.5</string></property>
         </item>
         <item>
          <property name="text"><string>2</string></property>
         </item>
        </widget>
      </item>
      <item>
        <widget class="QSpinBox" name="spinTimeout">
         <property name="minimum"><number>0</number></property>
         <property name="maximum"><number>60000</number></property>
         <property name="value"><number>1000</number></property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btnOpen">
         <property name="text"><string>Open</string></property>
        </widget>
      </item>
      <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text"><string>Close</string></property>
        </widget>
      </item>
     </layout>
    </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text"><string>Closed</string></property>
    </widget>
   </item>
  <item>
   <layout class="QHBoxLayout" name="statusRow">
    <item>
    <widget class="QProgressBar" name="progressActivity">
     <property name="minimum"><number>0</number></property>
     <property name="maximum"><number>100</number></property>
     <property name="value"><number>0</number></property>
    </widget>
    </item>
    <item>
    <widget class="QLabel" name="labelTimeout">
     <property name="text"><string>Timeout (ms)</string></property>
    </widget>
    </item>
   </layout>
  </item>
   <item>
    <widget class="QTextEdit" name="teRecv"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sendLayout">
     <item>
      <widget class="QLineEdit" name="editSend"/>
     </item>
     <item>
      <widget class="QCheckBox" name="chkAppendNewline">
       <property name="text"><string>Append newline</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text"><string>Send</string></property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="btnSaveAction"><property name="text"><string>Save</string></property></widget>
     </item>
  <item>
   <widget class="QLabel" name="labelLoadedAction"><property name="text"><string></string></property></widget>
  </item>
  <item>
   <widget class="QPushButton" name="btnClear"><property name="text"><string>Clear</string></property></widget>
  </item>
     <item>
      <widget class="QPushButton" name="btnLoadAction"><property name="text"><string>Load</string></property></widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
